charter.md

```diff
@@ -0,0 +1,49 @@
+# [DRAFT] WebViews Community Group Charter
+
+This Charter is work in progress. To submit feedback, please use [the GitHub repo](https://github.com/WebView-CG/charter) Issues where this chharter is being developed.
+
+## Goals
+
+The WebViews Community Group aims to reduce the issues arising from the use of software components (typically refered as `WebView`s) that are used to render Web technology-based content outside of a Web browser.
```

Comment:
- "refered" with the HTTP header? ;)
- Is it only about rendering? Are runtimes such as Deno and Node.js completely out of scope of the first phase?

The notion of "embedded web" was used during today's conversation. I wonder whether we could use it.

```suggestion
The WebViews Community Group aims to reduce issues arising from the use of software components (typicallly referred to as `WebView`s) that allow applications to embed Web technologies outside of a Web browser.
```

Reply:
I only put one "r" in referred, but you put 3 "l" in typically :)

I think we should limit the scope to WebViews (and thus rendering) - notwithstanding the broader web-native conversation we also want to start